packages/alias/alias.bench.mjs

```diff
@@ -0,0 +1,55 @@
+import { Suite, chartReport } from 'bench-node';
+import { execPath } from 'node:process';
+import { fileURLToPath } from 'node:url';
+
+import { spawnPromisified } from '../../utils/spawn-promisified.mjs';
+
+const suite = new Suite({
+	reporter: chartReport,
+});
+
+const cwd = fileURLToPath(import.meta.resolve('./fixtures'));
+const e2eTest = fileURLToPath(import.meta.resolve('./fixtures/e2e.mjs'));
+
+suite.add('--loader', { repeatSuite: 2 }, async () => {
+	await spawnPromisified(
```

Comment:
I'd change it to spawnSync to make  the bench suite faster

Reply:
https://github.com/nodejs-loaders/nodejs-loaders/pull/63